lib/agoneum/account/account.ex

```diff
@@ -63,7 +63,7 @@ defmodule Agoneum.Account do
   """
   def create_user(attrs \\ %{}) do
```

Comment:
I'd leave the `create_user` method as using the default changeset honestly, so that you can maybe use it for admin. I don't feel strongly about this.